_posts/2018-07-25-strimzi-0-5-0-release.md

```diff
@@ -0,0 +1,166 @@
+
+# Logging configuration improvements
+
+In order to improve the logging experience, it's now possible to specify the logging level (i.e. `DEBUG`, `INFO`, ...) in the Custom Resource describing the cluster both for Kafka and Zookeeper.
+There are two different ways of doing this, as showed by the following example.
+
+```yaml
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    ...
+    logging:
+      type: inline
+      loggers:
+        kafka.root.logger: "INFO,CONSOLE"
+    ...
+  zookeeper:
+    ...
+    logging:
+      type: external
+      name: customConfigMap
+    ...
+```
+
+The first specifies the log levels inline (that is, directly in the Kafka resource).
+
+The other way is to provide a reference to a complete log4j configuration stored externally in a ConfigMap.
+
+# Rack awareness
+
+In order to have high-availability, Apache Kafka provides a feature called "rack awareness".
+When your topics are replicated across multiple brokers, it is desirable for those brokers to be in different physical racks.
+If a rack fails, you are sure that other topic replicas are alive, a new leader can be elected and clients can continue to exchange messages. As opposite, if the brokers hosting replicas for a topic are put into the same rack, the partitions are not available anymore on rack failure because all brokers will go down at same time.
+
+The new Strimzi rack awareness feature provides a way to specify a `topologyKey` in order to spread brokers across different nodes.
+The `topologyKey` defines a node label which is used as `broker.rack` for configuring the Kafka brokers accordingly.
+
+```yaml
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+    rack:
+      topologyKey: failure-domain.beta.kubernetes.io/zone
+    # ...
+```
+
+When deploying the Kafka cluster in the public Cloud (i.e. Amazon, Azure and GCP), it means spreading brokers across different availability zones which can be compared to the physical racks.
+
+# Affinity and Tolerations
+
+Having Kafka brokers colocated on the same nodes with some other applications running on the same Kubernetes/OpenShift cluster can cause some issues like the _noisy neighbour_ phenomenon.
+This is really a problem when these applications are intensively utilizing the same resource - for example network bandwidth or disk I/O.
+
+Thanks to the support for affinity and tolerations in the 0.5.0 release, it's now possible to fix all the above problems.
+You'll get more information about this new feature in a dedicated blog post.
+
+# Cluster Operator handling RBAC
+
+The Cluster Operator is now in charge to handle RBAC resource so `ServiceAccount`, `RoleBinding` and `ClusterRoleBindings` for Kafka pods and for Topic Operator pods which need permissions for accessing resources (i.e. ConfigMap, Node, ...).
```

Comment:
"in charge of handling RBAC resources (ServiceAccount, RoleBinding and ClusterRoleBindings) ..."

Also, if you're monospacing K8s kinds here you should do so consistently thoughout the post.